web/src/app/modules/shared/components/presentation/button-group/button-group.component.ts

```diff
@@ -67,6 +83,52 @@ export class ButtonGroupComponent extends AbstractViewComponent<ButtonGroupView>
       this.buttonLoad.emit(true);
       this.doAction(payload);
     }
+    if (ref) {
+      if (isUrlExternal(ref)) {
```

Comment:
I think all of this special case logic can go away if we prefer using `LinkComponent` to wrap a button.

Reply:
Oddly enough, buttons are the exception to this because they are not components. Using the method in this PR, the entire button group would be wrapped as a link rather than individual buttons.

Reply:
AFAICT there is no good reason why Button isn't also a component type. It is just legacy and because "we never used them that way" .. but this is a good case for making button be stand-alone component that isn't tied to a group.

Reply:
Example: https://gist.github.com/wwitzel3/82fb4d1e0b6dad1fdb9bb204ff03214c

At least at first pass, it seem fine to make it a proper component. The current usage of button groups for components doesn't need to change, but it then allows folks to more generically use this new button component.